frontend/src/__tests__/cypress/cypress/support/commands/intercept-snapshots.ts

```diff
@@ -14,29 +14,6 @@ declare global {
   }
 }
 
-Cypress.Commands.add('visitWithLogin', (url) => {
```

Comment:
This was removed because it is a duplicate command which already exists in `application.ts`.